src/chapters/01/main.tex

```diff
@@ -109,15 +118,38 @@ \section{How to get the most out of this
 
 We feel that you can choose to read the book from cover to cover,
 writing out the code examples as you go; or it could also be used as a
-reference text when faced with particular problem and wanting to know
+reference text when faced with a particular problem and wanting to know
 where to start.
 
-The book is made up of 10 chapters that are paired in two 4 parts. Each
-part corresponds to a particular area of mathematics, for example
-``Emergent Behaviour''. Two chapters are paired together for each
-chapter, usually these two chapters correspond to the same area of
-mathematics but from a slightly different scale that correspond to
-different ways of tackling the problem.
+After this introductory chapter the book is split in to 4 sections.
+Each section corresponds to a broad problem type and contains 2 chapters that
+correspond to 2 solution approaches. The first chapter in a section is based on
+exact methodology whereas the second chapter is based on heuristic
```

Comment:
`exact` or `analytical`? I can't remember what we said. But this used to say `heuristic` which is wrong.